test/jest/__mock__/stripesComponents.mock.js

```diff
@@ -122,7 +151,24 @@ jest.mock('@folio/stripes/components', () => ({
       {children}
     </div>
   )),
-  Select: jest.fn(() => <div>Select</div>),
+  Select: ({ name, label, onChange, required, children, value }) => {
```

Comment:
I do not see the usage of this component inside `PickupServicePointFilter`. Do we really need to mock it?